monitoring/evidently/data-quality-report/data-quality-report-component.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kfp.components import create_component_from_func, InputPath, OutputPath"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BASE_IMAGE = \"quay.io/ibm/kubeflow-notebook-image-ppc64le:latest\"\n",
+    "\n",
+    "def data_quality_report(\n",
+    "    cur_path: InputPath('CSV'),\n",
+    "    output_path: OutputPath('HTML'),\n",
+    ") -> None:\n",
+    "\n",
+    "    import pandas as pd\n",
+    "    from evidently.metric_preset import DataQualityPreset\n",
+    "    from evidently.report import Report\n",
+    "    from pathlib import Path\n",
+    "    \n",
+    "    df = pd.read_csv(cur_path)\n",
+    "\n",
+    "    report = Report(metrics=[\n",
+    "        DataQualityPreset()\n",
+    "    ])\n",
+    "    \n",
+    "    report.run(current_data=df, reference_data=None, column_mapping=None)\n",
+    "    \n",
+    "    Path(output_path).parent.mkdir(parents=True, exist_ok=True)\n",
+    "    report.save_html(output_path)\n",
```

Comment:
Additionally render output as Visualization (as the HTML viewer would do as well)